src/bases/interface.ts

```diff
@@ -8,9 +8,10 @@ export interface BaseEncoder {
   /**
    * Base encodes to a **plain** (and not a multibase) string. Unlike
    * `encode` no multibase prefix is added.
+   *
    * @param bytes
    */
-  baseEncode(bytes: Uint8Array): string
+  baseEncode: (bytes: Uint8Array) => string
```

Comment:
Nit: I really dislike this eslint rule that reformat things this way. When I see new version I interpret it as `this` does not matter, while former version tells me you should call it as `instace.baseEncode`.

Reply:
not sure we have a choice if we're just wholesale importing this stuff via aegir do we? I don't feel too attached to the style here, it makes sense either way to my eyes but I do hate churn for churn's sake.